Classes/DataSources/RoleDataSource.php

```diff
@@ -61,13 +85,53 @@ public function getData(NodeInterface $node = null, array $arguments = [])
 
         foreach ($this->policyService->getRoles() as $role) {
             if (!$matchPatternArray($this->excludedPackages, $role->getPackageKey()) && !$matchPatternArray($this->excludedRoles, $role->getIdentifier())) {
-                $roles[$role->getIdentifier()] = [
-                    'label' => $role->getName(),
-                    'icon' => 'icon-users'
-                ];
+                $roles[$role->getIdentifier()] = $this->getDataByFullName($role->getPackageKey(), $role->getName());
             }
         }
 
         return $roles;
     }
-}
+
+    /**
+     * Get the translation and icon of a role by his name and package key
+     *
+     * @param string $packageName
+     * @param string $roleName
+     * @return array
+     */
+    protected function getDataByFullName(string $packageName, string $roleName): array
```

Comment:
Hey @EnesE, 

thank you very much for your PR! Having meaningful role names in the selector is an awesome improvement!

The objection that I have here, is that it adds a package special solution for a problem that is at least partially fixed in the core and in general should be fixed there. With Flow 7 I introduced label and description for roles and privileges (see: https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Configuration/Policy.yaml#L12). If you are already using Flow 7, it would be better to use these labels. What is missing, ist the translation of that labels.  That could be added here or added to the core for the next minor version.

Reply:
Hey @daniellienert ,
sorry for my late response.
I use this package in a customer project (Neos 5.1) where we needed german translations.

My change should make it possible to adjust the label for the role in the inspector as desired.
At the beginning we checked if the label is overwritten somewhere. If this is not the case, a translation should be checked in a central file. The default defined name is used as a fallback.